.npmrc

```diff
@@ -0,0 +1,14 @@
+# Enforce engine constraints
+engine-strict=true
+
+# Use frozen lockfile in CI and production
```

Comment:
This comment omits the fact that this setting is true for all developers too. The target of the setting is that the lockfile should never be updated unexpectedly, only when updating or running commands specifically with no-frozen-lockfile